bentoml/config/__init__.py

```diff
@@ -40,6 +40,15 @@ def expand_env_var(env_var):
             env_var = interpolated
 
 
+def parameterized_config(template):
+    """Generates a configuration from the provided template + variables defined in
+    current scope
+    :param template: a config content templated with {{variables}}
```

Comment:
Let's use the Google docstring style.  That way we can be consistent with rest of the project and generate good documentation.

Google docstring format:
"""Summary

Long Summary

Args:
    param 1:  description

Returns:
	type: description
Raises: